### PR TITLE
🐛 Correct links to specific experiments

### LINKED ISSUE #2138: Force experiment variant not working
## Description of the issue found on ampproject.org

Link: https://amp.dev/documentation/examples/components/amp-experiment/?format=websites

## How do we reproduce the issue?

Clicking any of the 3 links of the Development Tip tittle. 

What browser are you using? Google Chrome
What O/S are you using? Windows 10

### PR DESCRIPTION
**Pull request by matthiasrohmer**
Fixes #2138.

### CODE REVIEW

**Comment by matthiasrohmer** [comment]
Hah 😁You caught me, they're absolute now.

**Comment by sebastianbenz** [comment]
@matthiasrohmer are the additional query parameters needed?

**Comment by matthiasrohmer** [comment]
Even absolute links weren't enough to trigger a reload. You've got another idea?

**Comment by sebastianbenz** [comment]
`target=_blank`

**Comment by matthiasrohmer** [comment]
@sebastianbenz, just tested this. Interestingly this doesn't trigger the desired behaviour 😄With the `href` being the current location the anchor is then treated as it wouldn't have a `target` attribute.

File: examples/source/1.components/amp-experiment.html [hunk 1]
```diff
@@ -106,11 +106,10 @@
 <!-- ## Development tip -->
 <!--
   You can force an experiment to be in a specific variant by adding `amp-x-experiment-name` to the AMP page URL.
-  <ul>
-    <li><a href="<% hosts.platform %>/components/amp-experiment#amp-x-button-color-experiment=0">#amp-x-button-color-experiment=0</a></li>
-    <li><a href="<% hosts.platform %>/components/amp-experiment#amp-x-button-color-experiment=1">#amp-x-button-color-experiment=1</a></li>
-    <li><a href="<% hosts.platform %>/components/amp-experiment#amp-x-button-color-experiment=2">#amp-x-button-color-experiment=2</a></li>
-  </ul>
+
+  - [#amp-x-button-color-experiment=0](<% hosts.platform %>{{ doc.url.path }}?0#amp-x-button-color-experiment=0)
```
Comment (sebastianbenz) [inline examples/source/1.components/amp-experiment.html]: Does `target=_blank work` which I'd prefer?

Comment (matthiasrohmer) [reply examples/source/1.components/amp-experiment.html]: See the comment above: `target="_blank"` doesn't seem to open a new tab/window if it's the same location with a fragment.

Comment (sebastianbenz) [reply examples/source/1.components/amp-experiment.html]: 🤦‍♂️ sorry for missing your reply. Let's remove the get parameter in that case and add a comment to manually reload the page.

Comment (matthiasrohmer) [reply examples/source/1.components/amp-experiment.html]: All good, just added a hint!

**Pull request merged**